awx/ui/src/components/ScreenHeader/ScreenHeader.js

```diff
@@ -119,7 +119,11 @@ const Crumb = ({ breadcrumbConfig, showDivider }) => {
     <>
       {crumbElement}
       <Route path={`${match.url}/:path`}>
-        <Crumb breadcrumbConfig={breadcrumbConfig} showDivider />
+        <Crumb
+          breadcrumbConfig={breadcrumbConfig}
+          showDivider
+          data-cy={crumb}
```

Comment:
```suggestion
          data-cy=`${crumb}-crumb`
```
To avoid 2 elements with the same data-cy on the same component.